### PR TITLE
Fix admin sidebar on Mobile layout

### PR DESCRIPTION
**Pull request by tribela**
Sidebar toggle button was not function on mobile layout

### CODE REVIEW

File: app/javascript/flavours/glitch/packs/admin.js [hunk 1]
```diff
@@ -1,6 +1,27 @@
 import 'packs/public-path';
+import loadPolyfills from 'flavours/glitch/util/load_polyfills';
 import ready from 'flavours/glitch/util/ready';
 
+function setupSidebar() {
```
Comment (ClearlyClaire) [inline app/javascript/flavours/glitch/packs/admin.js:5]: Maybe rename it to `main` and move the `ready` call from below there?

Comment (tribela) [reply app/javascript/flavours/glitch/packs/admin.js:5]: Move it to the inside of ready function?

Comment (ClearlyClaire) [reply app/javascript/flavours/glitch/packs/admin.js:5]: No, the opposite: have `ready` be called by this (`setupSidebar`, which I propose should be renamed to `main`) function after loading polyfills, as other packs do.

Comment (tribela) [reply app/javascript/flavours/glitch/packs/admin.js]: OK, changed with 41396a7

**Comment by tribela** [comment]
@ClearlyClaire This is happening again

**Comment by ClearlyClaire** [comment]
Suspicion confirmed, fixing it in #1693!

**Pull request merged**